### ISSUE TITLE
[help] How to remove event listener?

### ISSUE THREAD

**Issue by Soggy-Pancake** [akinuri/js-lib#3]
I have the list of events and can filter them out to find the event listener I want but I can't figure out how to remove that event listener from the element. 

![image](https://github.com/akinuri/js-lib/assets/54160598/705825c8-b12b-4370-806f-4d0b82287389)

![image](https://github.com/akinuri/js-lib/assets/54160598/8216bc12-7681-40eb-9fc5-ef756b0328d6)

event is still there
![image](https://github.com/akinuri/js-lib/assets/54160598/3f4b1802-73a8-4049-97a3-181f78f65e0f)

**Comment by akinuri** [comment]
Can you provide a demo (e.g. a URL, the target element in the page, the target event/listener, etc.)? I can't debug using images :)

**Comment by Soggy-Pancake** [comment]
Ok my bad I can remove the event listeners I just can't find the right event listener. 

This is what I have and it seems to be working for removing some events so its at least working.
![image](https://github.com/akinuri/js-lib/assets/54160598/102c7ae8-72cf-48a5-af50-c85424cc7540)

The element I am trying to remove the event from is the download button under youtube videos.

![image](https://github.com/akinuri/js-lib/assets/54160598/61771f30-cdcc-4339-9374-f785cd103939)
![image](https://github.com/akinuri/js-lib/assets/54160598/3682173f-1810-406e-b7d5-50d19f5a194d)

**Comment by akinuri** [comment]
When I view a YouTube video (using Tampermonkey + this script), I see that the Download button has a single click event listener, and removing this manually (via the DevTools) disables the download action.

Now, when I select the button and run `EventListeners.get($0)`, I get an empty list. This is weird. However, when I inspect the button's parent (`yt-button-shape`), I see that it has two event listeners (`focusin` and `focusout`). I can also get these with `EventListeners.get($0)`.

Thinking that this might be a race condition issue, I dropped the `fetch()` usage and included the code in the tampermonkey script. Didn't change anything. Then I got more suspicious and timed the execution.

Adding `console.log(performance.now());` at the top of the script, I see that the tampermonkey script doesn't run immediately. I thought it would. It takes around 6 seconds for it to run. I'm not sure why it's not immediate and takes this long. If any event listeners are added during this time, they won't show up in the EventListeners object.

Might take a deeper look into this later, but currently, I can't think of a solution for this.
